### PR TITLE
PostersAndTalks

### PR DESCRIPTION
**Pull request by cherylepatrick**
Update to the publications page to add posters, talks and news articles. Note: I link to email addresses unless they are French as I am not sure of the legality of using a French email. The talks use the already-published slides from the conference websites, but for the posters I have just downloaded the latest from docDB - is that acceptable? Conference sites don’t tend to have them. Also I have introduced some colour as per collaboration meeting requests. I will use these colours on the rest of the site in due course.

### CODE REVIEW

**Comment by drbenmorgan** [comment]
Just two quick things 

1.  I'd suggest that all documentary content be stored in the Public area in DocDB. That makes it easy for the collaboration to contribute, and removed duplication. That'll need coordination with the Pub Board.

2. Remove the changes with formatting - colours etc. These should go in a separate PR.

**Comment by cherylepatrick** [comment]
Thanks @drbenmorgan, couple of questions: 

1) Totally agree for the longer term, but I can't do that myself. If someone else sorts out getting the stuff in the public docDB I'll be happy to update the links. Do you know who to contact for that? Karol?

2) How much do I need to take out? Eg can I keep the buttons if I just make them black? Or do you want all the buttons taken out and just the whole lot as a list?

Getting better at dealing with these multiple branches on the go at once, but still not great...

**Comment by cherylepatrick** [comment]
OK, the buttons are black now.

**Comment by drbenmorgan** [comment]
I'd guess Karol is the first port of call for 1. Probably also a good opportunity to get a placeholder on DocDB for approved plots?

Thanks for the changes on 2. I'll take a look as soon as I can!

On 8 Aug 2017, at 11:32, Cheryl Patrick <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks @drbenmorgan<https://github.com/drbenmorgan>, couple of questions:

  1.  Totally agree for the longer term, but I can't do that myself. If someone else sorts out getting the stuff in the public docDB I'll be happy to update the links. Do you know who to contact for that? Karol?

  2.  How much do I need to take out? Eg can I keep the buttons if I just make them black? Or do you want all the buttons taken out and just the whole lot as a list?

Getting better at dealing with these multiple branches on the go at once, but still not great...

File: _data/people.csv [hunk 1]
```diff
@@ -1,176 +1,147 @@
...
+Curtil,Christian,curtil@CPPM.IN2P3.FR,Curtil,Christian,,,Radon,,
 DAFNI,Theopisti,Theopisti.Dafni@CERN.CH,Dafni,Theopisti,,,,,y
-DARGAUD,Guillaume,dargaud@LPSC.IN2P3.FR,Dargaud,Guillaume,,,,,y
\ No newline at end of file
+DARGAUD,Guillaume,dargaud@LPSC.IN2P3.FR,Dargaud,Guillaume,,,,,
+Konovalov,Sergey,konovalov@itep.ru,Konovalov,Sergey,,,Source production,y,y
+Macko,Miroslav,macko@fmph.uniba.sk,Macko,Miroslav,PhD student,,,,y
+Palusova,Veronika,palusova@fmph.uniba.sk,Palusova,Veronika,PhD student,,,,y
+Mitra,Ankush,A.Mitra.2@warwick.ac.uk,Mitra,Ankush,,,,,y
+Morgan,Ben,Ben.Morgan@warwick.ac.uk,Morgan,Ben,Senior Research Fellow,assets/people/BenMorgan.jpg,Software development coordinator,y,y
```
Comment (goliviero) [inline _data/people.csv]: assets is missing

File: _data/people.csv [hunk 2]
```diff
@@ -1,176 +1,147 @@
...
+PERRY,Michael,michael.perry@manchester.ac.uk,Perry,Michael,Electronics Engineer,,,,y
+PIN,Axel,pin@cenbg.in2p3.fr,Pin,Axel,,,,,
+PIQUEMAL,Fabrice,piquemal@CENBG.IN2P3.FR,Piquemal,Fabrice,,,Joint spokesperson,,
+POINCHEVAL,Jerome,poincheval@LPCCAEN.IN2P3.FR,Poincheval,Jerome,,,,,
 POVINEC,Pavel,Pavel.Povinec@fmph.uniba.sk,Povinec,Pavel,,,,y,y
 PROGA,Marek,proga@PHYSICS.UTEXAS.EDU,Proga,Marek,,,,,y
-RAMACHERS,Yorck,y.a.ramachers@warwick.ac.uk,Ramachers,Yorck,,,Physics coordinator,y,y
-REMOTO,Alberto,alberto.remoto@lapp.in2p3.fr,Remoto,Alberto,,,,y,y
-RICHARDS,Ben,brichards@HEP.UCL.AC.UK,Richards,Ben,,,,,y
+RAMACHERS,Yorck,y.a.ramachers@warwick.ac.uk,Ramachers,Yorck,Faculty,assets/people/YorckRamachers.jpg,Physics coordinator,y,y
```
Comment (goliviero) [inline _data/people.csv]: assets is missing

File: _data/people.csv [hunk 3]
```diff
@@ -1,176 +1,147 @@
...
+Lieunard,Sophie,sophie.lieunard@lapp.in2p3.fr,Lieunard,Sophie,,,,,
 LIPTAK,Zachary,LIPTAK@PHYSICS.UTEXAS.EDU,Liptak,Zachary,,,,,y
-LIST,Serge,list@CENBG.IN2P3.FR,List,Serge,,,,,y
-LIU,Xin Ran,x.r.liu@UCL.AC.UK,Liu,Xin Ran,,,,,y
-LOAIZA,Pia,loaiza@LAL.IN2P3.FR,Loaiza,Pia,,,,y,y
-LUZON,Gloria,luzon@UNIZAR.ES,Luzon,Gloria,,,,,y
-LAMBERT,Fabian,fabian.lambert@LPSC.IN2P3.FR,Lambert,Fabian,,,,,y
-MAALMI,Jihane,maalmi@lal.in2p3.fr,Maalmi,Jihane,,,,,y
-MACKO,Miroslav,Miroslav.Macko@utef.cvut.cz,Macko,Miroslav,,,,,y
+LIU,Xi-Ran,x.r.liu@UCL.AC.UK,Liu,Xin-Ran,,assets/people/Xin_Ran_Liu_25x35w.jpg,,,y
```
Comment (goliviero) [inline _data/people.csv]: assets is missing

**Comment by goliviero** [comment]
We removed the posters files for the moment because they are large files. 
It must be discuss what is the politic about large (>1 MB) files in our repositories

**Comment by goliviero** [comment]
Travis CI failed but it was only because external links are slow and appears to time out. 
If we open links in our web browser, links work.

**Pull request merged**